core/src/main/java/com/klaytn/caver/wallet/KeyringContainer.java

```diff
@@ -116,9 +149,17 @@ public AbstractKeyring newKeyring(String address, String key) {
     /**
      * Creates a multiple type keyring instance with given parameters and add it to the keyringContainer.
      * KeyringContainer manages Keyring instance using Map {string:Keyring} which has address as key value.
+     * <pre>Example :
+     * {@code
+     * String address = "0x{address}";
+     * String[] keyArray = new String[] {caver.wallet.keyring.generateSingleKey(), caver.wallet.keyring.generateSingleKey(), ....};
```

Comment:
I think you can use `generateMultipleKeys`.

Reply:
I modified it.